custom-fonts.md

```diff
@@ -23,6 +29,22 @@ flutter:
         - asset: assets/fonts/abrilfatface/AbrilFatface-Regular.ttf
 ```
 
+Second use the font by delcaring the `TextStyle` constant and adding the font family used in the `pubsec.yaml`.
```

Comment:
Add a header here with "-ing" title. Like "Using the font in the app" or something